gui/GameConqueror.py

```diff
@@ -346,6 +346,14 @@ def __init__(self):
     
     # Memory editor
 
+    def MemoryEditor_Window_delete_event_cb(self, widget, event, data=None):
+        self.memoryeditor_window.hide()
+        return True
+    
+    def MemoryEditor_Close_button_clicked_cb(self, widget, data=None):
+        self.memoryeditor_window.hide()
+        return True
+    
```

Comment:
We have a general callback for this very common usecase, `hide_window_on_delete_event_cb`, use it to avoid code duplication.